org.adempiere.base/src/org/compiere/model/MWlistboxCustomization.java

```diff
@@ -0,0 +1,125 @@
+/******************************************************************************
+ * Product: iDempiere ERP & CRM Smart Business Solution                       *
+ * Copyright (C) 1999-2012 ComPiere, Inc. All Rights Reserved.                *
+ * This program is free software, you can redistribute it and/or modify it    *
+ * under the terms version 2 of the GNU General Public License as published   *
+ * by the Free Software Foundation. This program is distributed in the hope   *
+ * that it will be useful, but WITHOUT ANY WARRANTY, without even the implied *
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.           *
+ * See the GNU General Public License for more details.                       *
+ * You should have received a copy of the GNU General Public License along    *
+ * with this program, if not, write to the Free Software Foundation, Inc.,    *
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA.                     *
+ * For the text or an alternative of this public license, you may reach us    *
+ * ComPiere, Inc., 2620 Augustine Dr. #245, Santa Clara, CA 95054, USA        *
+ * or via info@compiere.org or http://www.compiere.org/license.html           *
+ *****************************************************************************/
+package org.compiere.model;
+
+import java.sql.ResultSet;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Properties;
+import org.compiere.util.Util;
+
+public class MWlistboxCustomization extends X_AD_Wlistbox_Customization {
+
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = -3220641197739038436L;
+
+	public MWlistboxCustomization(Properties ctx, int AD_Wlistbox_Customization_ID, String trxName) {
+		super(ctx, AD_Wlistbox_Customization_ID, trxName);
+		// TODO Auto-generated constructor stub
+	}
+
+	public MWlistboxCustomization(Properties ctx, ResultSet rs, String trxName) {
+		super(ctx, rs, trxName);
+		// TODO Auto-generated constructor stub
+	}
+	
+	public static MWlistboxCustomization get(Properties 	ctx, 
+											 int 			AD_User_ID, 
+											 String 		AD_WListboxName, 
+											 String 		trxName)
+	{
+		Query query = new Query(ctx, Table_Name, "AD_User_ID=? AND WListboxName=?", trxName);
+		return query.setClient_ID().setOnlyActiveRecords(true).setParameters(new Object[] { AD_User_ID, AD_WListboxName}).first();
```

Comment:
since the get here is AD_Client_ID, AD_User_ID and AD_WListboxName. The unique index for the table should reflect this, i.e should be this 3 column instead of AD_Client_ID, AD_Org_ID and AD_WListboxName

Reply:
Done